nautobot/docs/plugins/development.md

```diff
@@ -156,7 +156,7 @@ All required settings must be configured by the user. If a configuration paramet
 
 ### Install the Plugin for Development
 
-The plugin needs to be installed into the same python environment where Nautobot is, so that we can get access to `nautobot-server` command, and also so that the nautobot-server is aware of the new plugin.  
```

Comment:
This double space was intentional for the new paragraph.

Reply:
Not sure I understand your comment, sorry!

Reply:
The double space that was being removed on this line was intended to make a new paragraph (or a line break) - https://meta.stackoverflow.com/editing-help

Reply:
We already get a paragraph break from the blank line at line 160 - here's how this PR is rendering on RTD:

![image](https://user-images.githubusercontent.com/5603551/117326042-48b6d580-ae5f-11eb-87bb-97bdcc689e77.png)